website/docs/table-design/table-types/pk-table/merge-engines/default.md

```diff
@@ -0,0 +1,70 @@
+---
+sidebar_label: Default
+sidebar_position: 2
+---
+
+<!--
+ Copyright (c) 2025 Alibaba Group Holding Ltd.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+
+# Default Merge Engine
+
+## Overview
+
+The **Default Merge Engine** in Fluss retains the latest record for a given primary key. It supports the following operations: `INSERT`, `UPDATE`, `DELETE`.
+The Default Merge Engine is the default behavior in Fluss. It is enabled by default and does not require any additional configuration.
```

Comment:
Could you also mention how the default merge engine handles partial updates (keep last values for partial update columns)?  And also show examples about partial updates with reuslts.

Reply:
ok